ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2080,8 +2080,7 @@ tests:
     env:
       E2E_RUN_TAGS: '@fio'
       FILTERS_ADDITIONAL: ""
-      INDEX_IMAGE: quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-18@sha256:32eace7619c707671ad692c256a0cab5c89624cfe3c8a0eecd7874c566f56404
-      LVM_OPERATOR_SUB_CHANNEL: stable-4.19
```

Comment:
Removal of `LVM_OPERATOR_SUB_CHANNEL` is expected ?

Reply:
No, it should be a mistake. Thanks.